ramalama/cli.py

```diff
@@ -323,34 +324,31 @@ def logout_cli(args):
 def human_duration(d):
     if d < 1:
         return "Less than a second"
-    elif d == 1:
+    if d == 1:
         return "1 second"
-    elif d < 60:
+    if d < 60:
         return f"{d} seconds"
-    elif d < 120:
+    if d < 120:
         return "1 minute"
-    elif d < 3600:
+    if d < 3600:
         return f"{d // 60} minutes"
-    elif d < 7200:
+    if d < 7200:
         return "1 hour"
-    elif d < 86400:
+    if d < 86400:
         return f"{d // 3600} hours"
-    elif d < 172800:
+    if d < 172800:
         return "1 day"
-    elif d < 604800:
+    if d < 604800:
         return f"{d // 86400} days"
-    elif d < 1209600:
+    if d < 1209600:
         return "1 week"
-    elif d < 2419200:
+    if d < 2419200:
         return f"{d // 604800} weeks"
-    elif d < 4838400:
+    if d < 4838400:
         return "1 month"
-    elif d < 31536000:
+    if d < 31536000:
         return f"{d // 2419200} months"
-    elif d < 63072000:
-        return "1 year"
-    else:
-        return f"{d // 31536000} years"
+    return "1 year" if d < 63072000 else f"{d // 31536000} years"
```

Comment:
I think this is less clear to be honest.

Not sure about this linter to be honest, there's no clear advantage to A or B.

Maybe get rid of the else, so it's clear there's a guaranteed return